setup.py

```diff
@@ -1,15 +1,73 @@
 #!/usr/bin/env python
+import os
+import subprocess
+import sys
+from glob import glob
+from numpy import get_include as np_include
+from setuptools import setup, find_packages, Extension
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distribute_setup import use_setuptools
-    use_setuptools()
-    from setuptools import setup
+
+if os.path.exists('relic'):
+    sys.path.insert(1, 'relic')
+    import relic.release
+else:
+    try:
+        import relic.release
+    except ImportError:
+        try:
+            subprocess.check_call(['git', 'clone',
+                'https://github.com/jhunkeler/relic.git'])
+            sys.path.insert(1, 'relic')
+            import relic.release
+        except subprocess.CalledProcessError as e:
+            print(e)
+            exit(1)
+
+
+version = relic.release.get_info()
+relic.release.write_template(version, 'lib/pyraf')
 
 setup(
-    setup_requires=['d2to1>=0.2.3', 'stsci.distutils>=0.3.2'],
-    d2to1=True,
-    use_2to3=True,
-    zip_safe=False
+    name = 'pyraf',
+    version = version.pep386,
+    author = 'Rick White, Perry Greenfield, Chris Sontag',
+    author_email = 'help@stsci.edu',
+    description = 'Provides a Pythonic interface to IRAF that can be used in place of the existing IRAF CL',
+    url = 'https://github.com/spacetelescope/pyraf',
+    classifiers = [
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Topic :: Scientific/Engineering :: Astronomy',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+    ],
+    install_requires = [
+        'astropy',
```

Comment:
does not require astropy or nose or scipy or sphinx.  just numpy.  let's not bloat this to bigger than it has to be.